src/Parser/BitcoinMoneyParser.php

```diff
@@ -62,6 +62,20 @@ public function parse($money, $forceCurrency = null)
             $decimal = '0';
         }
 
-        return new Money($decimal, new Currency(BitcoinCurrencies::CODE));
+        if ($forceCurrency === null) {
+            $forceCurrency = new Currency(BitcoinCurrencies::CODE);
+        }
+
+        /*
+         * This conversion is only required whilst currency can be either a string or a
+         * Currency object.
+         */
+        $currency = $forceCurrency;
+        if (!$currency instanceof Currency) {
+            @trigger_error('Passing a currency as string is deprecated since 3.1 and will be removed in 4.0. Please pass a '.Currency::class.' instance instead.', E_USER_DEPRECATED);
+            $currency = new Currency($currency);
+        }
```

Comment:
I took this from 

https://github.com/moneyphp/money/blob/70b0c99beed1dc76dc2d6c7210df9e4f64db9159/src/Parser/DecimalMoneyParser.php#L49-L57